assets/src/web-stories-block/block/block-types/story-embed-block/embedControls.js

```diff
@@ -87,26 +81,11 @@ const EmbedControls = (props) => {
     <>
       <BlockControls>
         <ToolbarGroup>
```

Comment:
Do `wp.components.ToolbarGroup` and `wp.components.ToolbarButton` exist in 5.5 already?

The e2e test failures seem to indicate otherwise.